src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -112,39 +87,6 @@
     (.setCompileMode (get-compile-mode compile-mode))
     (.setEnvironment (managed-environment (get-system-env) gem-home))))
 
-(schema/defn ^:always-validate empty-scripting-container :- ScriptingContainer
-  "Creates a clean instance of a JRuby `ScriptingContainer` with no code loaded."
-  [ruby-load-path :- [schema/Str]
-   gem-home :- schema/Str
-   compile-mode :- jruby-schemas/SupportedJRubyCompileModes]
-  {:pre [(sequential? ruby-load-path)
-         (every? string? ruby-load-path)
-         (string? gem-home)]
-   :post [(instance? ScriptingContainer %)]}
-  (-> (ScriptingContainer. LocalContextScope/SINGLETHREAD)
-      (init-jruby-config ruby-load-path gem-home compile-mode)))
-
-(schema/defn ^:always-validate create-scripting-container :- ScriptingContainer
-  "Creates an instance of `org.jruby.embed.ScriptingContainer` and loads up the
-  puppet and facter code inside it."
-  [ruby-load-path :- [schema/Str]
-   gem-home :- schema/Str
-   compile-mode :- jruby-schemas/SupportedJRubyCompileModes]
-  ;; for information on other legal values for `LocalContextScope`, there
-  ;; is some documentation available in the JRuby source code; e.g.:
-  ;; https://github.com/jruby/jruby/blob/1.7.11/core/src/main/java/org/jruby/embed/LocalContextScope.java#L58
-  ;; I'm convinced that this is the safest and most reasonable value
-  ;; to use here, but we could potentially explore optimizations in the future.
-  (doto (empty-scripting-container ruby-load-path gem-home compile-mode)
-    ;; As of JRuby 1.7.20 (and the associated 'jruby-openssl' it pulls in),
-    ;; we need to explicitly require 'jar-dependencies' so that it is used
-    ;; to manage jar loading.  We do this so that we can instruct
-    ;; 'jar-dependencies' to not actually load any jars.  See the environment
-    ;; variable configuration in 'init-jruby-config' for more
-    ;; information.
-    (.runScriptlet "require 'jar-dependencies'")
-    (.runScriptlet "require 'puppet/server/master'")))
-
 (schema/defn ^:always-validate config->puppet-config :- HashMap
```

Comment:
I think this might end up being the only function still needed in this namespace, in which case maybe just move it into `jruby-puppet-core`?